.github/workflows/labeler.yml

```diff
@@ -0,0 +1,13 @@
+name: "Pull Request Labeler"
+on:
+  pull_request_target:
```

Comment:
Might it be an idea to also add a trigger for [issues](https://docs.github.com/en/actions/reference/events-that-trigger-workflows#issues)?

Reply:
Good idea, @gerrymanoim, does that work right now?  I think I would like to try this out and see how it goes.

Reply:
Note that with the issue template applying labels already, this may not be that useful for issues.

Reply:
Right, I forgot about that, so BUG and ENH are already there, and most of the others apply much less. Lets just try it out, it seems like it is probably nice. Thanks @gerrymanoim!

Reply:
Thanks! I hope it is helpful. I've never tested issues, but could have a look at that.